jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -305,7 +327,8 @@ boolean isRPCServer() {
               startTimeMicroseconds,
               startTimeNanoTicks,
               computeDurationViaNanoTicks,
-              tags);
+              tags,
+              references);
```

Comment:
Thrift Span model would be cleaner without a direct parentId field. If the `parentId` has been designed to save a little bit of payload then it should be used. 

I think it should be only used to represent `childOf` relationship. Otherwise, it's ambiguous. 

What if we establish a convention that `parentId` (in thrift span) should be used only when there is one parent as `childOf`. Otherwise, use only list of references. Thoughts?